ikasaneip/visualisation/dashboard/src/main/java/org/ikasan/dashboard/ui/general/component/ProgressIndicatorDialog.java

```diff
@@ -46,7 +46,7 @@ public void open(String label)
         layout.setHorizontalComponentAlignment(FlexComponent.Alignment.CENTER, progressBar);
         layout.setSizeFull();
 
-        this.setSizeFull();
+//        this.setSizeFull();
```

Comment:
No size?

Reply:
The change in the way the dialog works automatically sizes to the content. It used to be the case that setting size full did that but it now makes it take the entire screen,

Reply:
I'll remove the commented out code.